### PR TITLE
[7] Develop a Contact form page using HTML

### PR DESCRIPTION
**Pull request by Egwolo2**
## My HTML Forms`
- `I added my html forms folder which includes my contact page, login page and registration page.

image
image
image

### CODE REVIEW

**Comment by Egwolo2** [comment]
okay. 
Noted Tolu.

**Comment by Egwolo2** [comment]
image

File: submissions/egwoloOkorodudu/Html/Contact.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <input type="email" name="email" id="email"/><br><br/>
+            <label for="subject">Subject</label><br>
+            <input type="text" name="subject" id="subject"/><br><br/>
+
+            <label for="message">Message</label><br>
+            <textarea name="messag" id="message" cols="40" rows="10"></textarea><br><br/> 
+
+            <button onclick="submit">Submit</button><br><br/>
+    
+
```
Comment (ToluAkin) [inline submissions/egwoloOkorodudu/Html/Contact.html:25]: Remove extra spaces here

Comment (Egwolo2) [reply submissions/egwoloOkorodudu/Html/Contact.html:24]: Changes made @ToluAkin

**Pull request merged**